Control relationships

lucasm-iRonin opened the pull request:
This PR contains changes from [this PR](https://github.com/fotinakis/jsonapi-serializers/pull/62) + fixes I mentioned in the comments.

I've also added tests to make sure it's working as expected.

fotinakis commented on spec/serializer_spec.rb:
```diff
@@ -352,6 +352,74 @@ def serialize_primary(object, options = {})
+      })
+    end
+
+    it 'allows to inclue data for relationships' do
```
"inclue" :)

fotinakis commented on spec/support/serializers.rb:
```diff
@@ -113,6 +112,38 @@ def meta
+    include JSONAPI::Serializer
+
+    attribute :title
+    attribute :long_content do
```
Consider dropping the `long_content` attribute from both of these serializers and the tests, since we don't care about testing this behavior here (it's tested elsewhere) so it's kind of just noise and could be excluded.

fotinakis commented:
This looks great! The tests definitely make me happy.

Any chance you'd like to add a small update to the `README` to document these? I can do it if not.

lucasm-iRonin commented:
@fotinakis thank you for your feedback. I've updated specs according to your suggestions and fixed the typo. I've also updated `README` file.

Please check when you have a moment. Thanks!

fotinakis commented:
Thanks for your contribution!

This has been released in v0.11.0 of the gem.
